platform-includes/getting-started-capture-errors/javascript.hono.mdx

```diff
@@ -1,34 +1,5 @@
 ### Report Unhandled Exceptions
 
-Next, bind an `onError` hook to report unhandled exceptions to Sentry:
+Sentry automatically reports exceptions reported by the `onError` function from Hono. In case the error comes with a status code, it captures all errors except for the ones with a 3xx or 4xx status code.
```

Comment:
l: maybe it is worth to mention that this is only true, unless it gets overwritten by `shouldHandleError`. Or the other way around to write inside `shouldHandleError` that this will overwrite the auto reports from Sentry

Reply:
I think I'll just add a "By default, " at the beginning of the sentence, since I am referring to customizing the behavior below.